runtime/doc/eval.txt

```diff
@@ -6152,7 +6155,9 @@ synstack({lnum}, {col})					*synstack()*
 		valid positions.
 
 system({expr} [, {input}])				*system()* *E677*
```

Comment:
perhaps should change first argument to `{cmd}` as it works the same as the `{cmd}` in `jobstart`. Even if the explanation is repeated, I think it also should point out that it works the same as `jobstart`. This way, it is easier to reason if it is a "safe" refactor to change between the functions.